src/main/java/net/anotheria/anoprise/dataspace/persistence/DataspaceNotFoundException.java

```diff
@@ -8,11 +8,6 @@
 public class DataspaceNotFoundException extends DataspacePersistenceServiceException {
 
 	/**
-	 * Basic serialVersionUID variable.
-	 */
-	private static final long serialVersionUID = 4675725133571302930L;
-
-	/**
 	 * Public constructor.
```

Comment:
>Because this is exception is not serialized and deserialized anywhere.
How do you know? Have you checked all projects that are using this service? 
I would even say that your company has used this service as distributed service some time ago, and therefore has serialized / deserialized this exact exception...